server/uploadHandler/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "web_template",
+  "version": "1.0.0",
+  "author": "SAM CLI",
+  "license": "MIT",
```

Comment:
Please update the name, author and license

Reply:
what should I change this to ?

Reply:
Remove the license or change it to an empty string. Maybe change the name to "upload_hander" and change the author to "VTAA"?